.github/stable/external.txt

```diff
@@ -109,8 +109,7 @@ packaging==24.0
 pandocfilters==1.5.1
 parso==0.8.4
 pathspec==0.12.1
-PennyLane-Catalyst==0.5.0
-PennyLane_Lightning==0.35.1
+PennyLane_Lightning==0.36.0
```

Comment:
We need catalyst reintroduced here, but this should likely happen in a forked version of the RC branch, rather than making edits directly.